presto-testng-services/src/main/java/io/prestosql/testng/services/SuppressTestngChecks.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.testng.services;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+import static java.lang.annotation.ElementType.METHOD;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+@Retention(RUNTIME)
+@Target(METHOD)
+public @interface SuppressTestngChecks
```

Comment:
Maybe name it to reflect it suppresses method annotation checks (and not eg test class naming)

Reply:
What do you suggest? I was thinking about that I didn't find anything better? `@SuppressNonTestMethodCheck`?

Reply:
Maybe just `@ReportUnannotatedMethods.Suppress` (with the intention not to import that class directly) ?

Reply:
Good idea, thanks.